.github/workflows/run-tests.yml

```diff
@@ -2,21 +2,26 @@ name: Run Tests
 on: workflow_call
 
 jobs:
-  unit_tests:
+  studio:
     strategy:
       matrix:
         os: [windows-latest, ubuntu-latest]
-        node-version: [18.x]
```

Comment:
For my own edification, why is the `node-version` no longer needed?

Reply:
we default to 18 in setup-ci